### PR TITLE
Create Contributing.md

### PR DESCRIPTION
**Pull request by znicholls**

### CODE REVIEW

**Comment by znicholls** [comment]
@chrisroadmap would be great to get your thoughts on whether `Contributing.md` is feasible or just ridiculous

**Comment by rgieseke** [comment]
I think this needs something on how to adress issues in their original repos and where to find them.

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+The repository contains a `.editorconfig` file. This ensures that all of our text editors behave the same way and avoids spurious changes simply due to differing whitespace or end of line rules.
+
+Many editors have built in support for `Editorconfig` but some require a plugin. To work out if your editor requires a plugin, please check https://editorconfig.org/.
+
+## Getting started
+
+### Your First Contribution
+
+The development methodology for MAGICC makes heavy use of `git`, `make`, virtual environments and test driven development. If you aren't familiar
```
[the remark this answers is not in the document]
Comment (znicholls) [reply Contributing.md]: I don't know what you're talking about...

**Comment by znicholls** [comment]
> I think this needs something on how to adress issues in their original repos and where to find them.

What do you mean?

**Comment by rgieseke** [comment]
> > I think this needs something on how to adress issues in their original repos and where to find them.

> What do you mean?

If there is a problem with Pymagicc, try reproducing it with vanilla Pymagicc and file an issue at the Pymagicc repo.

**Comment by rgieseke** [comment]
General thoughts, i think atm it's too specific and claims a bunch of things we don't have (yet) or won't have ... (kind of tests etc.)

In the beginning it will probably be just us, so writing down how to develop, review PRs etc. is probably enough?

**Comment by znicholls** [comment]
> General thoughts, i think atm it's too specific and claims a bunch of things we don't have (yet) or won't have ... (kind of tests etc.)

In my opinion the kinds of test are a super important thing to get your head around (especially mocking) if we want to build this well. Hence I'd prefer to leave it in there. Even if we don't have them yet, reading it in the contributing guide makes you think about it which is a good step in the right direction.

**Pull request merged**